stable/admin/templates/statefulset.yaml

```diff
@@ -75,8 +75,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
```

Comment:
I suggest it would be better to do all the concatenation in the {{template}} in the _helpers.yaml file.

Reply:
I've done this but kept the cluster domain definition separate still, with the thought it might be needed in the future on its own